xeeg_hed_score/sub-eegSeizureTUH/ses-eeg01/eeg/sub-eegSeizureTUH_ses-eeg01_task-rest_run-000_events.tsv

```diff
@@ -0,0 +1,2 @@
+onset	duration	HED	channel
+0	234	sc:Other-organized-rhythm/non-interesting events	FP1-F7,F7-T3,T3-T5,T5-O1,FP2-F8,F8-T4,T4-T6,T6-O2,A1-T3,T3-C3,C3-CZ,CZ-C4,C4-T4,T4-A2,FP1-F3,F3-C3,C3-P3,P3-O1,FP2-F4,F4-C4,C4-P4,P4-O2
```

Comment:
the channel columns should probably only have the first channel name in it, because the channels.tsv file already lists the reference channel